.github/workflows/update.yml

```diff
@@ -0,0 +1,60 @@
+name: Update README and LaTeX Build File
+
+on:
+  workflow_dispatch:
+
+jobs:
+  update-files:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
```

Comment:
What version do we get here? Should we be more explicit?

Reply:
I will use 3.12.2